### PR TITLE
Clean up - remove old colors - Stage 4

### PR DESCRIPTION
**Pull request by damianmef**

### CODE REVIEW

File: lib/styles/modifiers/_colors.scss [hunk 1]
```diff
@@ -1,35 +1,31 @@
...
 
-.-fireflyBlack {
-	color: $color-firefly-black;
-}
-
-.-delugeViolet {
-	color: $color-deluge-violet;
+.-violet500 {
+	color: var(--raw-violet-500);
 }
```
Comment (kvas-damian) [inline lib/styles/modifiers/_colors.scss:31]: I believe 1 class is missing. Was it intended?

Comment (damianmef) [reply lib/styles/modifiers/_colors.scss:31]: This class has no usage in DS and in Platform.

Comment (damianmef) [reply lib/styles/modifiers/_colors.scss:31]: @kvas-damian what do You think if we will remove all these classes? I will be better `clean up`. I checked usages in the Platform:
```
-gray400 - 5
-gray700 - 19
-gray300 - 4
-white - 0?
-green500 - 4
-blue500 - 0
-black  - 0
-violet500 - 0
```
All cases: 32.

I think we should replace like this:
```
<td class="-gray400">
```
to:
```
<td class="grayCell"> 
and
.gray { var(--raw-gray-400); }
```

File: lib/styles/settings/_shadows.scss [hunk 2]
```diff
@@ -1,6 +1,15 @@
-@import 'colors';
 @import 'colors/tokens';
 
+$color-shadow: rgba(var(--raw-black-rgb), 0.3);
+$color-success-shadow: rgba(var(--raw-green-500-rgb), 0.7);
+$color-danger-shadow: rgba(var(--raw-red-500-rgb), 0.7);
+$color-warning-shadow: rgba(var(--raw-gold-500-rgb), 0.7);
+
+// Tokens - shadows
+$color-neutrals-shadow-weak: $color-default-shadow-weak;
```
Comment (kvas-damian) [inline lib/styles/settings/_shadows.scss:9]: why do we have to create an alias on this variable? Isn't original name good enough?

Comment (damianmef) [reply lib/styles/settings/_shadows.scss]: I think I didn't want to touch shadows, it was before shadows migration. Now these variables have no usages, so I removed them.

File: lib/styles/modifiers/_colors.scss (hunk 1 above)
Comment (damianmef) [reply lib/styles/modifiers/_colors.scss]: File removed.

**Review by kvas-damian — APPROVED** [review]

**Pull request merged**